omniledger/service/api.go

```diff
@@ -168,6 +172,31 @@ func (c *Client) GetChainConfig() (*ChainConfig, error) {
 	return config, nil
 }
 
+// WaitProof will poll omniledger until a given instanceID exists.
+// It will return the proof of the instance created. If value is
+// non-nil, it will wait for the value of the proof to be equal to
+// the value.
```

Comment:
No comment on timeout behaviour. I was expecting maybe a special errTimeout val returned, but in fact as implemented timeout = valid *Proof with !Match() and err == nil.

Reply:
and didn't use the `value` - both fixed now.